storage/remote/dial_context.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
Not that it really matters legally, but we usually put the current year here when creating new files.

Reply:
Fixed